src/Core/src/Exception/Http/HttpExceptionTrait.php

```diff
@@ -58,6 +61,8 @@ public function __construct(ResponseInterface $response)
             if (0 === stripos($h, 'content-type:')) {
                 if (preg_match('/\bjson\b/i', $h)) {
                     $isJson = true;
+                } elseif (preg_match('/\bxml\b/i', $h)) {
```

Comment:
I'm not sure it's a good thing. sometimes AWS does not respect the content type in response :-(.

It can help when present, but when missing we should add something to guess it'

Reply:
Okey. So I look for json and fallback to XML